src/components/validation-form/index.tsx

```diff
@@ -39,23 +39,34 @@ const ValidationForm: FC<Props> = ({
   expand,
   includeExpandedTriples,
   shapesCollection,
-  validatorActions: { fetchShapesCollectionRequested: fetchShapesCollection },
+  ontologyCollection,
+  validatorActions: {
+    fetchShapesCollectionRequested: fetchShapesCollection,
+    fetchOntologyCollectionRequested: fetchOntologyCollection
+  },
   isLoading,
   onValidate,
   ...props
 }) => {
   const shapesOptions = shapesCollection
-    ? shapesCollection.shapes.map(shapesDef => ({
+    ? shapesCollection.map(shapesDef => ({
```

Comment:
Kan ta bort ternary her:

```suggestion
    shapes?.map(shapesDef => ({
```